src/main/java/com/hamarb123/macos_input_fixes/FabricReflectionHelper.java

```diff
@@ -959,6 +960,47 @@ public static String StringVisitable_getString(Object instance)
 	}
 	private static MethodHandle _StringVisitable_getStringMethod;
 
+	/**
+	 * <p>Intemediary name: {@code method_68564}</p>
+	 * <p>Mapped name: {@code getStringOr}</p>
```

Comment:
The actual name is still `getString` (https://maven.fabricmc.net/docs/yarn-1.21.5+build.1/net/minecraft/nbt/NbtCompound.html#getString(java.lang.String,java.lang.String))